src/block.ts

```diff
@@ -90,17 +119,19 @@ class Block {
       // console.log(`Merkle root is valid`)
     } else if (this.transactions) {
       for (const transaction of this.transactions) {
+        // TODO: Needs an index passed in
         this.addMerkleHash(transaction.getHash());
       }
     } else {
       throw new Error(`Must call addMerkleHash on all transactions first`);
     }
   }
```

Comment:
@kevinejohn  `addMerkleHash` requires an index as the first argument, but one wasn't being passed in. Not sure what this should be